pages/faq.js

```diff
@@ -104,24 +105,22 @@ const PlaceHolderText = styled.td`
 export default function Faq({ hackathons }) {
   // remove'LHD2021' when integrated with sidebar to receive hackathon that is passed down
   const [faqs, setFaqs] = useState([]);
-  const [newFaq, setNewFaq] = useState({});
+  const [newFaq, setNewFaq] = useState({ hackathonIDs: [] });
   const [faqViewing, setFaqViewing] = useState({});
   const [faqEditing, setFaqEditing] = useState({});
   const [faqConfirm, setFaqConfirm] = useState({});
   const [addNew, setAddNew] = useState(false);
   const [isLoading, setIsLoading] = useState(true);
   const [alertMsg, setAlertMsg] = useState('');
   const { email: user } = useAuth().user;
+
+  const fetchFaqs = async () => {
+    const faqsFetched = await getFaqs();
+    if (Object.keys(faqsFetched).length > 0) setFaqs(faqsFetched);
+  };
   useEffect(() => {
-    setIsLoading(true);
-    getFaqs().then((res) => {
-      if (Object.keys(res).length > 0) {
-        setFaqs(res);
-      } else {
-        setFaqs([]);
-      }
-      setIsLoading(false);
-    });
+    fetchFaqs();
```

Comment:
this should be awaited and this useffect should be async

Reply:
My bad, I actually created an async `fetchFaqs` helper and didn't push my changes. Is this ok now?
```
const fetchFaqs = async () => {
    const faqsFetched = await getFaqs();
    if (Object.keys(faqsFetched).length > 0) setFaqs(faqsFetched);
  };
```

Reply:
you should still await that function before setting loading to false though so when calling fetchFaqs you'd write await fetchFaqs

Reply:
sometimes you get react warnings when trying to use async inside useEffect, maybe we can setLoading(false) inside fetchFaqs?